src/api/modelTypes.js

```diff
@@ -145,49 +108,155 @@ export type User = $ReadOnly<{|
    */
   avatar_url: AvatarURL,
 
-  // These properties appear in data from the server, but we ignore
-  // them. If we add these, we should try to avoid `avatar_url`
-  // falling out of sync with them.
-  // avatar_source: mixed,
-  // avatar_url_medium: mixed,
-  // avatar_version: mixed,
-
-  // profile_data added in commit 02b845336 (in 1.8.0);
-  // see also e3aed0f7b (in 2.0.0)
-  // (This one doesn't appear in `/users` responses.)
-  profile_data?: empty, // When we need this, describe its actual type.
+  // If we use this, avoid `avatar_url` falling out of sync with it.
+  -avatar_version: number,
+
+  full_name: string,
+  is_admin: boolean,
+
+  // TODO(server-3.0): New in FL 8
+  is_owner?: boolean,
+
+  // TODO(server-5.0): New in FL 73
+  is_billing_admin?: boolean,
+
+  // TODO(server-4.0): New in FL 59
+  // Possible values are:
+  //   Organization owner => 100
+  //   Organization administrator => 200
+  //   Organization moderator => 300
+  //   Member => 400
+  //   Guest => 600
+  role?: number,
```

Comment:
In principle I'd say the same thing about this as about `bot_type`: leave these details to just be in the API docs. But maybe the role is an important enough property to make an exception.

Reply:
Probably fine to leave these details out. It doesn't say anything surprising or counterintuitive, and it seems like the kind of thing where the API doc might change (for example, to add new roles, or to stop using `=>`, where `:` would do fine and can't be confused with `≥`).